src/Integration/Directory.Build.props

```diff
@@ -0,0 +1 @@
+<Project><PropertyGroup><IntegrationVersion>3.0.0-fixpackaging0001</IntegrationVersion></PropertyGroup></Project>
```

Comment:
I've now released that for this work the way I intended it, we probably don't want to check this in, since it will always be changing.